docs/rest_api2.rst

```diff
@@ -0,0 +1,4259 @@
+|                    | {                                                                                                                                                                           |
+|                    |                                                                                                                                                                             |
+|                    | "interfaces": [                                                                                                                                                             |
+|                    |                                                                                                                                                                             |
+|                    | {                                                                                                                                                                           |
+|                    |                                                                                                                                                                             |
+|                    | "path": "Pod-1/Node-101/eth1/1",                                                                                                                                            |
+|                    |                                                                                                                                                                             |
+|                    | "name": "internal"                                                                                                                                                          |
+|                    |                                                                                                                                                                             |
+|                    | },                                                                                                                                                                          |
+|                    |                                                                                                                                                                             |
+|                    | {                                                                                                                                                                           |
+|                    |                                                                                                                                                                             |
+|                    | "path": "Pod-1/Node-101/eth1/2",                                                                                                                        
```

Comment:
This is used for assigning and unassigning VLANs.  ... then it deletes the VLAN from the guest BIG-IP, otherwise